Stresser/RegistryArtifact.cpp

```diff
@@ -0,0 +1,43 @@
+#include "RegistryArtifact.h"
+
+RegistryArtifact::RegistryArtifact(std::string name, std::string data) : IArtifact(name, data)
+{
+	this->registryPrefix = RegistryArtifactUtils::getRegistryPrefix(this->data);
+	this->registrySubKey = RegistryArtifactUtils::getRegistrySubKey(this->data);
+	this->install();
+}
+
+RegistryArtifact::~RegistryArtifact()
+{
+	this->uninstall();
+}
+
+void RegistryArtifact::install() const
+{
+	HKEY hKey;
+
+	auto createRegStatus = RegCreateKeyExA(
+		this->registryPrefix, 			// Registry key
+		this->registrySubKey.c_str(), 	// Sub key
+		0, 								// Reserved and must be 0
+		NULL, 							// Class type of the key
+		REG_OPTION_NON_VOLATILE,		// Keep the key after reboot
+		KEY_WRITE, 						// Registry key security and access right
+		NULL, 							// Security attributes
+		&hKey, 							// Handle to the opened key
+		NULL 							// Determine either the key exists or not
+	);
+
+	AutoRegistryKeyHandle autoHKey(hKey);
+
+	if (createRegStatus != ERROR_SUCCESS) 
```

Comment:
Replace the order: first the constant value (`ERROR_SUCESS`)  and then the variable.
This is called YODA notation and it's for better safety (to prevent code mistakes the compiler couldn't find in compile time).

My convention is to add { } always, with new line :-)